MAUI/Pull-to-Refresh/customization.md

```diff
@@ -0,0 +1,640 @@
+            this.progressbar.HeightRequest = 55;
+            this.progressbar.IndeterminateAnimationDuration = 750;
+            this.progressbar.Content = this.progressContent;
+
+            bindable.BindingContext = this.viewModel;
+            this.pullToRefresh = bindable.FindByName<Syncfusion.Maui.PullToRefresh.SfPullToRefresh>("pullToRefresh");
+            this.dataGrid = bindable.FindByName<SfDataGrid>("dataGrid");
+            this.dataGrid.ItemsSource = this.viewModel.OrdersInfo;
+            this.pullToRefresh.Refreshing += this.PullToRefresh_Refreshing;
+            this.pullToRefresh.Pulling += this.PullToRefresh_Pulling;
+
+            var pullingTemplate = new DataTemplate(() =>
+            {
+                return new ViewCell { View = this.frame };
+            });
+
+            this.pullToRefresh.PullingViewTemplate = pullingTemplate;
+            this.pullToRefresh.RefreshingViewTemplate = pullingTemplate;
+
+            base.OnAttachedTo(bindable);
+        }
+
+        private void PullToRefresh_Pulling(object? sender, PullingEventArgs e)
+        {
+            this.progressbar!.TrackThickness = 0.8;
+            this.progressbar.TrackRadiusFactor = 0.1;
+            this.progressbar.IsIndeterminate = false;
+            this.progressbar.ProgressFill = Color.FromRgb(0, 124, 238);
+            this.progressbar.TrackFill = Colors.White;
+
+            var absoluteProgress = Convert.ToInt32(Math.Abs(e.Progress));
+            this.progressbar.Progress = absoluteProgress;
+            this.progressbar.SetProgress(absoluteProgress, 1, Easing.CubicInOut);
+            this.progressContent!.Text = e.Progress.ToString();
+        }
+
+        private async void PullToRefresh_Refreshing(object? sender, EventArgs e)
+        {
+            this.progressContent!.IsVisible = false;
+            this.pullToRefresh!.IsRefreshing = true;
+            await Task.Delay(10);
+            await this.AnimateRefresh();
+            this.viewModel!.RefreshItemsource(10);
+            await Task.Delay(10);
+            this.pullToRefresh.IsRefreshing = false;
+            this.progressContent.IsVisible = true;
+        }
+
+        private async Task AnimateRefresh()
+        {
+            this.progressbar!.Progress = 0;
+            this.progressbar.IsIndeterminate = true;
+
+            await Task.Delay(750);
+            this.progressbar.ProgressFill = Colors.Red;
+
+            await Task.Delay(750);
+            this.progressbar.ProgressFill = Colors.Green;
+
+            await Task.Delay(750);
+            this.progressbar.ProgressFill = Colors.Orange;
+
+            await Task.Delay(750);
+        }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![PullToRefresh view Template](Images/customization/template-slideontop.gif)
+
+You can download the sample code by clicking the following link: [Sample](https://github.com/SyncfusionExamples/load-datagrid-as-pullable-content-of-.net-maui-pull-to-refresh).
```

Comment:
add the sample link in Notes like below (Refer Expander GettingStarted UG]

![image](https://github.com/syncfusion-content/maui-docs/assets/44426156/7b658a9c-cad1-4d08-888b-9a917abc1322)

Reply:
changed